File: Contest-07/A.py

def max_final_score(n, nums):
 
    nums.sort()
    score = 0

    for i in range(n):
  
        score += min(nums[i], nums[i + n])
    return score

t = int(input())


for _ in range(t):

    n = int(input())
    nums = list(map(int, input().split()))

    print(max_final_score(n, nums))

File: Contest-07/A.cpp
#include<iostream>
#include<vector>
using namespace std;

int main()
{
    int n;

cin >> n;

vector<int> numbers(2* n);

for (int i = 0; i < 2* n; i++) {

cin >> numbers[i];

} sort(numbers.begin(), numbers.end());

int score = 0;

for (int i = 0; i < 2 * n; i += 2) {

score += numbers[i];

}

cout << score << endl;
}